### PR TITLE
Ports Opossums from Desert Rose

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request

Ports opossums from desert rose for a bounty, gib money pls.

dreamseeker_pFbBa4hImU

Every map except Cogstation has had them mapped in. I am unable to edit Cogstation due to errors about dicebags or something.

As I am not familiar with 90% of the maps, I simply placed Poppy the Safety Opossum in places near engineering, or in maint bars, while a single regular opossum has been placed on the opposite end in maint from where Poppy is.

dreamseeker_Da8SD6SuER

## Changelog
:cl: Toriate
add: Opossums have migrated into the maintenance tunnels! Seek them out at your own peril!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/possum.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	gold_core_spawnable = FRIENDLY_SPAWN
+
+/mob/living/simple_animal/opossum/poppy
+	name = "Poppy the Safety Possum"
+	desc = "Safety first!"
+	icon_state = "poppypossum"
+	icon_living = "poppypossum"
+	icon_dead = "poppypossum_dead"
+	butcher_results = list(/obj/item/clothing/head/hardhat = 1)
```
Comment (timothyteakettle) [inline code/modules/mob/living/simple_animal/friendly/possum.dm]: you should put a newline here
also good pr

**Comment by Tupinambis** [comment]
10/10 PR. Not adding them to cog is actually a sin though.  
  
Finally the maint rework we were all waiting for...

File: _maps/map_files/KiloStation/KiloStation.dmm [hunk 2]
```diff
@@ -71314,7 +71314,7 @@
 	min_oxy = 5;
 	name = "Lia";
 	real_name = "Lia";
-	response_help = "pets";
+	!INVALID_VAR! = "pets";
```
Comment (Hatterhat) [inline _maps/map_files/KiloStation/KiloStation.dmm]: uhhhhhhhhhhhhhhhhhhhhhhh

**Comment by Michiyamenotehifunana** [comment]
Newline has been added to possum.dm, Kilostation's invalid var (Ostensibly from outdated var overrides, there's a longer fancier name for it now) has been reverted back to what it was, and I still can't figure out how to open CogStation without cocking everything up.

**Comment by EmeraldSundisk** [comment]
If you can get Cog working, you have my go-ahead to turn Rick into an opossum (as opposed to the rat he currently is).

**Pull request merged**